include/stomp_moveit/stomp_moveit_planning_context.hpp

```diff
@@ -28,5 +28,8 @@ class StompPlanningContext : public planning_interface::PlanningContext
 private:
   const stomp_moveit::Params params_;
   std::shared_ptr<stomp::Stomp> stomp_;
+
+  bool extractSeedTrajectory(const planning_interface::MotionPlanRequest &req,
```

Comment:
looks like this could be a free function instead. I'm also thinking about changing this to return std::optional in the future, but this version works for me as well